src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -45,13 +44,10 @@
 #include "common/encoding.hpp"
 #include "common/instance.hpp"
 #include "common/new.hpp"
-#include "common/settings.hpp"
 #include "lib/platform/exit_code.h"
 #include "lib/spinel/radio_spinel.hpp"
 #include "lib/spinel/spinel.h"
 #include "lib/spinel/spinel_decoder.hpp"
-#include "meshcop/dataset.hpp"
-#include "meshcop/meshcop_tlvs.hpp"
```

Comment:
I am happy that we remove this. Specially that we no longer have dependency in `radio_spinel` module to these core internal headers `meshcop_tlvs.hpp` and `dataset.hpp` :)